clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -237,7 +237,7 @@ private void process(final ConsumerRebalanceListenerCallbackNeededEvent event) {
     private final int defaultApiTimeoutMs;
     private final boolean autoCommitEnabled;
     private volatile boolean closed = false;
-    private final Optional<ClientTelemetryReporter> clientTelemetryReporter;
+    private Optional<ClientTelemetryReporter> clientTelemetryReporter = Optional.empty();
```

Comment:
Could you please add comments to explain why it needs initial value?

Reply:
Fixed